packages/kbn-config/README.md

```diff
@@ -0,0 +1,3 @@
+# `@kbn/config` — Kibana configuration file loader
+
+This package contains the configuration service used to load and read the kibana configuration file
```

Comment:
Can you include some basic usage information here?

Reply:
Yea, in the follow-up, as there is currently no way to properly use the configuration service that has been extracted. (This is only step 1 of https://github.com/elastic/kibana/issues/76003, PR would have been too big otherwise)